### DISCUSSION TITLE
Search Working Unexpected

### DISCUSSION THREAD

**Discussion by kaganergovan** [elrumo/macOS_Big_Sur_icons_replacements#1203] [Help with the website] [answered]
Hello Dear Community,
I can't search MYKI and uTorrent web (or Torrent) icons. I got "No results" when I type any letter.

Regards,

**Comment by elrumo** [comment, marked as answer]
That was my bad, I accidentally removed the lines needed to connect to the search backend. Should be fixed now.

**Reply by kaganergovan** [reply]
Thank you so much, fixed right now. ✌🏻
